administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -178,13 +178,28 @@ export class ApplicationViewComponent extends React.Component<{
   }
 }
 
+const sortByStatus = (a: number, b: number): number => a - b
+const sortByDateAsc = (a: Date, b: Date): number => a.getTime() - b.getTime()
+
+const getVerificationStatus = (status: number[]): VerificationStatus => {
```

Comment:
I feel like it would be better to do define the function like
```ts
const getApplicationStatus = (application: Application): ApplicationStatus => {
```
and introduce a new `ApplicationStatus` enum (which would currently hold the same entries as VerificationStatus).

Reply:
yes i already thought about that. :)

Reply:
i renamed them for better understanding to `fullyRejected`, `fullyApproved`, `ambiguous`